docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -140,14 +140,19 @@ Allows setting a sort order for the result. Defaults to `start_time`, i.e. sorti
 (Optional, string)
 Sort order. Valid values are `asc` for ascending and `desc` for descending order. Defaults to `asc`, meaning ascending order.
 
+`after_value`::
```

Comment:
I wonder if we should call the parameter `after_sort_value` to make the coupling to sort explicit in the name?

I am still pondering on `after_`. Reading your text, perhaps `start_sort_value` is better, though I am not really contend with that either...

Reply:
Not sure about this or the alternatives. How about just `from` maybe? It's short and it indicates "start" + "inclusive"?